Rakefile

```diff
@@ -1,7 +1,34 @@
+# frozen_string_literal: true
+
 require 'bundler/gem_tasks'
 require 'rspec/core/rake_task'
 
+assets_target = 'app/assets'
+js_target = "#{assets_target}/javascripts/guide"
+css_target = "#{assets_target}/stylesheets/guide"
+
+CLEAN << assets_target
 RSpec::Core::RakeTask.new(:spec)
+task spec: %w[generate_assets]
+task build: %w[clean generate_assets]
```

Comment:
Configure the `build` task (that builds the gem) to first clean the `app/assets` directory and regenerate the assets.